.poshchan/settings.json

```diff
@@ -1,4 +1,9 @@
 {
+    "version": "0.1",
+    "azdevops": {
+        "organization": "powershell",
```

Comment:
This is not required as I default to using the same org/project as the GitHub repo.  But it doesn't hurt to be explicit.  However, if we keep this, might as well have correct casing for PowerShell here too :)